### PR TITLE
LNS+ SDK 1.0 support

### PR DESCRIPTION
**Pull request by cbouvet-ledger**
tests ok

### CODE REVIEW

File: src/emulate_lnsp_1.0.c [hunk 1]
```diff
@@ -0,0 +1,322 @@
+#define _SDK_2_0_
```
[the remark this answers is not in the document]
Comment (greenknot) [reply src/emulate_lnsp_1.0.c:1]: I'm not sure about this file, let's talk about it offline.

**Review by greenknot — CHANGES_REQUESTED** [review]

**Review by greenknot — CHANGES_REQUESTED** [review]
I made a few minor comments, if you can address them before merging it'd be great. Thanks for the PR!

File: tests/apps/conftest.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 def app_info_from_path(path) -> AppInfo:
     # name example: nanos#btc#1.5#5b6693b8.elf
-    app_regexp = re.compile(r"^(nanos|nanox|blue)#([^#]+)#([^#][\d\w\-.]+)#([a-f0-9]{8})\.elf$")
```
[the remark this answers is not in the document]
Comment (Saltari) [reply tests/apps/conftest.py:16]: done

File: src/emulate_lnsp_1.0.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (Saltari) [reply src/emulate_lnsp_1.0.c:1]: it is used actually in the syscalls header to know if we use the old syscall system with ID_IN/ID_OUT or the the new one.
it does not compile without this define

**Review by greenknot — APPROVED** [review]

**Pull request merged**